WORKSPACE

```diff
@@ -389,9 +371,9 @@ container_pull(
 # TODO build fedora_with_test_tooling for multi-arch
 container_pull(
     name = "fedora_with_test_tooling",
-    digest = "sha256:ce36d2b4f81b038fba0b61b1bb1ac7f671d47687fb1f9d7ddedd22742cc79dd9",
     registry = "quay.io",
-    repository = "kubevirtci/fedora-with-test-tooling",
+    repository = "rmohr/fedora-with-test-tooling",
```

Comment:
I tried `docker pull rmohr/fedora-with-test-tooling:fc32-2107231046` in my server but failed.

Reply:
@zhlhahaha yes sorry made a mistake when pushing. It is available now.

Reply:
> @zhlhahaha yes sorry made a mistake when pushing. It is available now.

I still get the following error message, if I run `docker pull rmohr/fedora-with-test-tooling:fc32-2107231046`
"Error response from daemon: pull access denied for rmohr/fedora-with-test-tooling, repository does not exist or may require 'docker login': denied: requested access to the resource is denied"

But it works if I add `quay.io` prefix as `docker pull quay.io/rmohr/fedora-with-test-tooling:fc32-2107231046`

Reply:
Shouldn't the image be located in `quay.io/kubevirtci` repo?

Reply:
Yes, but first I have to make sure that the new image will work. Until that is known it should not be in the official location.